lib/restart.js

```diff
@@ -33,8 +33,8 @@ function checkRoot(callback) {
     let data = '';
 
     const child = require('child_process').spawn('whoami', []);
-    child.stdout.on('data', text => data += text.toString().replace('\n', ''));
-    child.stderr.on('data', text => data += text.toString().replace('\n', ''));
+    child.stdout.on('data', text => data += text.toString().replace(/\n/g, ''));

```

Comment:
I assume we want to replace all occurences, not only the first?

Reply:
think so

Reply:
yes